src/darwin/frida-helper-backend-glue.m

```diff
@@ -216,6 +217,12 @@ static gboolean frida_agent_context_init_functions (FridaAgentContext * self, Gu
 static kern_return_t frida_set_debug_state (mach_port_t thread, gconstpointer state, GumCpuType cpu_type);
 static void frida_set_hardware_breakpoint (gpointer state, GumAddress break_at, GumCpuType cpu_type);
 
+static gboolean frida_find_libc (const GumModuleDetails * details, gpointer user_data);
+static GumAddress frida_find_libc_initializer (guint task, GumAddress base);
+static GumAddress frida_get_module_slide (gconstpointer command, gsize ncmds, GumAddress base);
```

Comment:
This one is quite low-level compared to the others so I'd put it right after `frida_create_capstone()`.